components/button/overview.md

```diff
@@ -32,14 +32,19 @@ To add a Telerik Button to your Blazor app, use the `<TelerikButton>` tag:
 ````CSHTML
 @result
 <br />
-<TelerikButton OnClick="@OnClickHandler">Hello!</TelerikButton>
+@moreInfo
+
+<br />
+<TelerikButton OnClick="@OnClickHandler">Click me!</TelerikButton>
 
 @code {
     string result;
+    string moreInfo;
 
-    async Task OnClickHandler()
+    async Task OnClickHandler(MouseEventArgs args)
```

Comment:
Consider adding an example somewhere (either a new example in the events section, or revert the one on overview) that shows handling without arguments so that people know they can.

Reply:
Done, examples in the overview pages reverted.